metrics/report/report_dockerfile/scaling.R

```diff
@@ -139,26 +201,28 @@ mem_line_plot <- ggplot() +
 # If we only have relatively few samples, add points to the plot. Otherwise, skip as
 # the plot becomes far too noisy
 if ( skip_points == 0 ) {
-	mem_line_plot = mem_line_plot + geom_point( data=data, aes(count, avail_gb, colour=testname, group=dataset), alpha=0.3)
+	mem_line_plot = mem_line_plot + geom_point(alpha=0.3)
 }
 
 cpu_stats_plot = suppressWarnings(ggtexttable(data.frame(cstats),
 	theme=ttheme(base_size=10),
 	rows=NULL
 	))
 
-# plot how samples varioed over  'time'
-cpu_line_plot <- ggplot() +
-	geom_line( data=data, aes(count, cpu_idle, colour=testname, group=dataset), alpha=0.2) +
-	geom_smooth( data=data, aes(count, cpu_idle, colour=testname, group=dataset), se=FALSE, method="loess", size=0.3) +
+# plot how samples varied over  'time'
+cpu_line_plot <- ggplot(data=fndata, aes(as.numeric(as.character(pod)), 
```

Comment:
moving this to the base ggplot removes unnecessary duplication

Reply:
yep. as above, iirc, I did have some instance in the past where it did not work out for me (possibly having multiple data sets of variants of plots on the same diagram). But, am fine to dedup where we can :-)